src/styles/blocks/card.scss

```
@@ -0,0 +1,90 @@
+.card {
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: $card-width;
+  padding: 16px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: $basic-margin-top-for-card-elements auto 0;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 40px 0 0;
```

just margin-bottom fix in all cases

```suggestion
    margin-top: 40px;
```